src/app/common/utils.ts

```diff
@@ -44,15 +44,20 @@ interface MakeTxExplorerLinkArgs {
   mode: BitcoinNetworkModes;
   suffix?: string;
   txid: string;
+  bitcoin: BitcoinChainConfig;
```

Comment:
Not sure I understand passing in a `bitcoin` prop here when this is a shared helper function across chains and using a switch statement? 🤔

Reply:
The switch case uses the 'mode' property, which is either mainnet or testnet. But if the user is using regtest (which is also testnet in mode), it will try to fetch the regtest tx from a testnet explorer (which will not exist on that chain). So i was proposing to pass in the bitcoin object to check if the network is regtest, and if so, use the endpoint of that network to fetch the tx.